packages/actions/src/helpers/utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Get a value from a key information about a circuit.
+ * @param circuitInfo <string> - the stringified content of the .r1cs file.
+ * @param rgx <RegExp> - regular expression to match the key.
+ * @returns <string>
+ */
+export const getCircuitMetadataFromR1csFile = (circuitInfo: string, rgx: RegExp): string   => {
+    // Match.
+    const matchInfo = circuitInfo.match(rgx)
+
+    if (!matchInfo) return ''
```

Comment:
Give a short explanation for this error.

Bonus: use a code (e.g., throw new Error('S3-001: short explanation here')).